src/managers/StageInstanceManager.js

```diff
@@ -28,31 +28,30 @@ class StageInstanceManager extends BaseManager {
 
   /**
    * Options used to create a stage instance.
-   * @typedef {Object} CreateStageInstanceOptions
-   * @property {StageChannel|Snowflake} channel The stage channel whose instance is to be created
+   * @typedef {Object} StageInstanceCreateOptions
    * @property {string} topic The topic of the stage instance
    * @property {PrivacyLevel|number} [privacyLevel] The privacy level of the stage instance
    */
 
   /**
    * Creates a new stage instance.
-   * @param {CreateStageInstanceOptions} options The options to create the stage instance
+   * @param {StageChannel|Snowflake} channel The stage channel to associate the created stage instance to
+   * @param {StageInstanceCreateOptions} options The options to create the stage instance
    * @returns {Promise<StageInstance>}
    * @example
    * // Create a stage instance
-   * guild.stageInstances.create({
-   *  channel: '1234567890123456789',
+   * guild.stageInstances.create('1234567890123456789', {
    *  topic: 'A very creative topic',
    *  privacyLevel: 'GUILD_ONLY'
    * })
    *  .then(stageInstance => console.log(stageInstance))
    *  .catch(console.error);
    */
-  async create(options) {
-    if (typeof options !== 'object') throw new TypeError('INVALID_TYPE', 'options', 'object', true);
-    let { channel, topic, privacyLevel } = options;
+  async create(channel, options) {
```

Comment:
Same as kyra, I do not understand why channel is now outside of the options object, and personally I'd prefer it be a full object. Thoughts @discordjs/the-big-3?

Reply:
The unified approach is consistent with [`RoleManager`](https://discord.js.org/#/docs/main/master/class/RoleManager?scrollTo=create), [`ThreadManager`](https://discord.js.org/#/docs/main/master/class/ThreadManager?scrollTo=create), and a few more places. Although I think in some places, `create` is split in two, such as [`GuildManager`](https://discord.js.org/#/docs/main/master/class/GuildManager?scrollTo=create), [`GuildChannelManager`](https://discord.js.org/#/docs/main/master/class/GuildChannelManager?scrollTo=create), and [`GuildBanManager`](https://discord.js.org/#/docs/main/master/class/GuildBanManager?scrollTo=create).

There's also one with three arguments: [`GuildEmojiManager`](https://discord.js.org/#/docs/main/master/class/GuildEmojiManager?scrollTo=create).

Something needs to be done in a later PR to merge those in objects and increase the consistency across the library, to be honest.

Reply:
I seperated `channel` because then the typedef `StageInstanceCreateOptions` can be reused in `StageChannel#createStageInstance`. The other option which Kyra suggested was to keep the `channel` but that would render it useless for the said method as it eventually gets set to `this.id` inside it. `ReplyMessageOptions` was created to address exactly this for `Message#reply`. Are we going to go back to the same thing again?

If we do want to have a single param here then I think we can create a new typedef for `options` param in `StageChannel#createStageInstance` that will not have the `channel` property.

Reply:
We discussed this internally, @iCrawl says to keep them separated, and if we want to, we can merge arguments in a later PR.